### ISSUE TITLE
1

### ISSUE THREAD

**Issue by ArtemGrr** [AligatorVodaema/Credit-calculator#1]
https://github.com/AligatorVodaema/Credit-calculator/blob/7a1cf2a66a3b6184f2122bac359c1ba65e912fe1/serializers.py#L46

тут и без квадратных скобок сработает, разберись, почему

**Comment by vsevolodGolangDev** [comment]
да, знаю почему. типо будет считаться как генераторное выражение.
